dist/adapter.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
```

Comment:
Please remove directory dist/*
You don't need save compiled js code from ts code

Reply:
Don't understand you

Reply:
It's not correct to push compiled and builded code. Your builded code have been saved in directory **dist**.